android/src/main/java/com/it_nomads/fluttersecurestorage/ciphers/RSACipher18Implementation.java

```diff
@@ -168,6 +186,6 @@ private void createKeys(Context context) throws Exception {
             kpGenerator.initialize(spec);
             kpGenerator.generateKeyPair();
         }
+        setLocale(localeBeforeFakingEnglishLocale);
```

Comment:
@mogol Just wondering if this should be encapsulated in a finally block instead? Would this have any side effects if an exception was thrown and this wasn't called?

Reply:
yes, it makes sense to do so just in case...